### DISCUSSION TITLE
License Management & Advanced Security

### DISCUSSION THREAD

**Discussion by Atlas-LinhVN** [community/community#135872] [Enterprise]
### Select Topic Area

Question

### Body

Hello guys,

I, Owner role, currently have 1 Organization and 1 Team with 6 members, 4/6 members already have Pro licenses. Can you please advise the minimum license & type for the Owner to have Advance Security add-on to scan the code created by the members.

My purposes are:

Managing the team that members have Pro licenses

All codes created by members must be scanned (vulnerability, security)

Owner
1 x Enterprise license
1 x Advanced security license
Enterprise license is only needed for the Owner?
Question: Can the Owner use Advanced Security to scan the code created by all members?

Team: members x Pro license.
Question: Pro license is enough for my organization?
Many thanks

**Comment by queenofcorgis** [comment]
Hi @Atlas-LinhVN ,

The GitHub Community on Discussions is a space for all GitHub users to come and discuss ideas and projects. Your post was closed because we already moved your initial discussion to the Enterprise category. As a result, we have closed the duplicate post, and we ask that you no longer cross-post.

For additional guidance, please review our [Community on Discussions Code of Conduct](https://github.com/community/community/blob/main/CODE_OF_CONDUCT.md) and [GitHub’s Terms of Service](https://docs.github.com/en/github/site-policy/github-terms-of-service).
